src/app/Main/Body/QueryPage/QueryPageContent/Parameter/QueryPage-ParameterContent.component.tsx

```diff
@@ -33,18 +84,33 @@ function createParameterList(
                 parameterLabel = t(`Subfilters.${filterElement}.name`);
             }
 
-            const parameterListItem = createParameterListItemService(
-                filterElement,
-                selectedFilters[filterElement],
-                t,
-                isSubFilter
-            );
+            const listItems: JSX.Element[] = [];
+            selectedFilters[filterElement].forEach((parameter) => {
+                const parameterListItem = createParameterName(
+                    filterElement,
+                    parameter,
+                    t,
+                    isSubFilter
+                );
+                listItems.push(
+                    <ListItem
```

Comment:
Reject: ListItems and the corresponding List are still splitted over two components. You could just use the parameterName components as input for the ParameterListLayout and compose the List there.

Reply:
Yes, because I hadn't edited that part yet. Now it is done.